internals-js/src/federation.ts

```diff
@@ -567,7 +585,9 @@ export class FederationMetadata {
       this.extendsDirective(),
     ];
     return this.isFed2Schema()
-      ? baseDirectives.concat(this.shareableDirective())
+      ? baseDirectives
+        .concat(this.shareableDirective())
```

Comment:
Nit: `concat` accepts a list of arguments so it can be done with a single concat (not that it matters, and happy to keep as is if you prefer, just wondered if this was an oversight).